frontend/src/components/Main.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect } from "react";
+import { useSelector } from "react-redux";
+import { useNavigate, Link } from "react-router-dom";
+import { useDispatch } from "react-redux";
+import { API_URL } from "utils/utils";
+import thoughts from "reducers/thoughts";
+
+const Main = () => {
+  const accessToken = useSelector((store) => store.user.accessToken);
+  const thoughtItems = useSelector((store) => store.thoughts.items);
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+
+  useEffect(() => {
```

Comment:
I wonder if this useEffect might be causing the issue with your link to login.

Reply:
Agree with Sofia. I believe that, maybe, since you don't seem to have a "sign out"-button (that I can see) the accessToken never disappears and so when you try to go back to login it takes you straight back to the main since you're still logged in, if that makes sense?